submissions/alstep07/memory-pair-game/app.js

```diff
@@ -0,0 +1,126 @@
+const main = document.querySelector(".main");
+const game = {
+	count: 0,
+	moves: 0,
+	currentPair: [],
+	frontImgSrc: "./img/front.png",
+	getRandomIds() {
+		let ids = [0, 1, 2, 3, 4, 5];
```

Comment:
It looks like this array does not need to be recreated on every call of `getRandomIds()` method. It is not much of an overhead at this scale, but still better to keep IDs as a constant outside of method.